lib/new_relic/agent/instrumentation/view_component/instrumentation.rb

```diff
@@ -14,8 +14,6 @@ def render_in_with_tracing(*args)
           name: metric_name(self.class.identifier, self.class.name)
         )
         yield
-      rescue => e
-        ::NewRelic::Agent.logger.debug('Error capturing ViewComponent segment', e)
```

Comment:
Hi @mjacobus! Thank you for catching this—we shoudn't be swallowing errors! But we do want to capture and report them to New Relic. Could you update your code to the following?

```ruby
rescue => e
  NewRelic::Agent.notice_error(e)
  raise
```

Then we can get this merged ◡̈

Reply:
Sure thing! There you go. Thank you @hannahramadan